test/Kestrel.Interop.TestSites/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+ using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Extensions.Logging;
+
+namespace Kestrel.Interop.TestSites
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var hostBuilder = new WebHostBuilder()
+                .ConfigureLogging((_, logging) =>
+                {
+                    logging.AddConsole();
+                    logging.SetMinimumLevel(LogLevel.Trace);
+                })
+                .UseKestrel(options =>
+                    options.ListenAnyIP(0, listenOptions =>
+                    {
+                        listenOptions.Protocols = HttpProtocols.Http2;
+                        listenOptions.UseHttps(TestResources.GetTestCertificate());
+                    }))
+                .Configure(app =>
+                    app.Run(async (context) =>
+                    {
+                        if (string.Equals(context.Request.Query["TestMethod"], "POST", StringComparison.OrdinalIgnoreCase))
+                        {
+                            await context.Response.SendFileAsync("post.html");
```

Comment:
Using a small html page to force chrome to send a POST instead of GET

Reply:
You can inline this HTML, no need for the file.